.gitignore

```diff
@@ -8,6 +8,8 @@ doc/*.css
 doc/*.html
 doc/*.png
 doc/edoc-info
-.directory
-.eunit
-.rebar
+rebar3
+rebar.lock
```

Comment:
This file is typically checked in.

Reply:
Why should it be?  PropEr does not have any dependencies, let alone locked ones, and it seems that `rebar3` automatically generates this `rebar.lock` file when run.  What I am missing?